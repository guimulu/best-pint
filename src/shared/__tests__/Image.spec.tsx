import * as React from "react";
import { createSnapshot } from "jest-react-fela";

import { Image } from "..";

describe("Image component", () => {
  it("renders correctly", () => {
    const snapshot = createSnapshot(
      <Image
        alt="test"
        src="/test.png"
        marginBottom={10}
        marginTop={10}
        maxHeight={10}
      />
    );
    expect(snapshot).toMatchSnapshot();
  });
});
