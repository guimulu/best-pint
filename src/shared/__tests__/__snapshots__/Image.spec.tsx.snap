// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image component renders correctly 1`] = `
".a {
  max-height: 10px;
}
.b {
  margin-top: 10px;
}
.c {
  margin-bottom: 10px;
}
.d {
  width: auto;
}


<img src=/test.png alt=test className=a b c d />;
"
`;
